tests/core/test_sqlalchemy_config.py

```diff
@@ -77,7 +77,7 @@ def test_sql_alchemy_connect_args(
         with conf_vars(config):
             settings.configure_orm()
             engine_args = {}
-            if settings.SQL_ALCHEMY_CONN.startswith('mysql'):
+            if settings.SQL_ALCHEMY_CONN.startswith('mysql') or settings.SQL_ALCHEMY_CONN.startswith('mssql'):
```

Comment:
```suggestion
            if settings.SQL_ALCHEMY_CONN.startswith(('mysql', 'mssql')):
```

Since `startswith()` can accept a tuple of strings

Reply:
TIL! Thanks. i had no idea, it's just one sentence in the docs mentioning it :)